cmd/release-controller/http_helper.go

```diff
@@ -238,6 +244,63 @@ func upgradeCells(upgrades *ReleaseUpgrades, index int) string {
 	return buf.String()
 }
 
+func upgradeJobs(upgrades *ReleaseUpgrades, index int, tagCreationTimestampString string) string {
+	buf := &bytes.Buffer{}
+	buf.WriteString(`<td>`)
+	// filter out upgrade job results more than 24 hours old because we don't want
+	// buildcops looking at them.
+	cutoff := metav1.NewTime(time.Now().Add(-24 * time.Hour))
+
+	t, e := time.Parse(time.RFC3339, tagCreationTimestampString)
+	if e != nil {
+		// if we can't parse the date, assume it's current/new.
+		t = time.Now()
+	}
+	tagCreationTimestamp := metav1.NewTime(t)
+	if (&tagCreationTimestamp).Before(&cutoff) {
+		buf.WriteString(`</td><td></td><td>skipped reporting based on age of tag`)
```

Comment:
This is too much text on the page and is visually confusing - can you simply have it be empty?  People on this dashboard are looking for failures, I don't think the message adds anything when scanning and forces a user to think about details they don't care about.  Or `<em>--</em>`